test/e2e/utils_test.go

```diff
@@ -658,3 +662,75 @@ func verifyWorkPropagationAndMarkAsApplied(memberClusterName, crpName string, re
 		}, eventuallyDuration, eventuallyInterval).Should(Succeed())
 	}
 }
+
+func buildTaints(memberClusterNames []string) []clusterv1beta1.Taint {
+	var labels map[string]string
+	taints := make([]clusterv1beta1.Taint, len(memberClusterNames))
+	for i, name := range memberClusterNames {
+		labels = labelsByClusterName[name]
+		taints[i].Key = labels[regionLabelName]
+		taints[i].Value = labels[envLabelName]
```

Comment:
Hi Arvind! The taint's key is the region label name, but the value is using an env label name. Is this expected?

Reply:
Same for the other few occurrences below.

Reply:
Oh, I know that the form is correct; it's just normally the k/v pair should be regionLabel <-> regionValue (a label key and its value) rather than regionLabel <-> envLabel (two label keys).